mkat_fpga_tests/test_cbf.py

```diff
@@ -654,20 +653,21 @@ def test_sensor_values(self):
 
             # Varify the number of sensors received with
             # number of sensors in the list.
-            self.assertTrue((values_reply.reply_ok() == list_reply.reply_ok())
-                , msg='Sensors Failure: {}'
-                .format(roach.host))
+            Aqf.is_true((values_reply.reply_ok() == list_reply.reply_ok())
+                , '{}: Varify the number of sensors received with number of '
```

Comment:
I assume that should be Verify, not Varify? But perhaps a better description would be:

Verify that ?sensor-list and ?sensor-value agree about the number of sensors.